### PR TITLE
Painkiller addition and adjustment

### PR DESCRIPTION
**Pull request by Urgan123**
Description
I added pain killer values to alcohol per their written Strength
so strong drinks like vodka will relieve 50 pain
weaker drinks like ale will only relieve 20
for reference paracetamol is 130 pain 
also changed the values of paracetamol and tramadol for paracetamol needed tramadol to make it but tramadol had the higher pain relievence value

### CODE REVIEW

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 1]
```diff
@@ -2144,3 +2438,14 @@
 	glass_name = "special blend whiskey"
 	glass_desc = "Just when you thought regular station whiskey was good... This silky, amber goodness has to come along and ruin everything."
 	glass_center_of_mass = list("x"=16, "y"=12)
+
+/datum/reagent/ethanol/specialwhiskey/affect_ingest(var/mob/living/carbon/M, var/alien, var/effect_multiplier)
+    ..()
+    M.add_chemical_effect(CE_PAINKILLER, 50)
+
```
Comment (TorinoFermic) [inline code/modules/reagents/reagents/food-Drinks.dm]: That's tons of newlines.

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 2]
```diff
@@ -1047,11 +1056,12 @@
 	glass_icon_state = "beerglass"
 	glass_name = "beer"
 	glass_desc = "A freezing pint of beer"
-	glass_center_of_mass = list("x"=16, "y"=8)
+	glass_center_of_mass = list("x"=16, "y"=8)
```
Comment (TorinoFermic) [inline code/modules/reagents/reagents/food-Drinks.dm]: Accidental space at the end of line

File: code/modules/reagents/reagents/food-Drinks.dm (hunk 1 above)
Comment (TorinoFermic) [reply code/modules/reagents/reagents/food-Drinks.dm]: Wait, they are spaces not tabs behind ..() and M.add.....

Comment (Urgan123) [reply code/modules/reagents/reagents/food-Drinks.dm]: Thanks for that ill do that

Comment (TorinoFermic) [reply code/modules/reagents/reagents/food-Drinks.dm]: image
Here's from Notepad++

Comment (Urgan123) [reply code/modules/reagents/reagents/food-Drinks.dm]: Okay everything should work now

**Comment by TorinoFermic** [comment]
Last thing before I merge, where I can find the 130 pain reduction from paracetamol ?

This line is from code/modules/reagents/reagents/medicine.dm
> /datum/reagent/medicine/paracetamol/affect_blood(var/mob/living/carbon/M, var/alien, var/effect_multiplier)
> 	M.add_chemical_effect(CE_PAINKILLER, 50)

**Comment by Urgan123** [comment]
Right i did a big dumm and didint save the para fix but if i would have it would mess up traitor kits so ill do it in another project and tweak the traitor kits if thats okay with you

**Comment by TorinoFermic** [comment]
You could do but it will create a merge conflict to be resolved later here after a merge from other PR.

**Comment by Urgan123** [comment]
okay everything should work now

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 3]
```diff
@@ -302,6 +302,19 @@
 	new /obj/item/weapon/reagent_containers/pill/tramadol(src)
 	new /obj/item/weapon/reagent_containers/pill/tramadol(src)
 
+/obj/item/weapon/storage/pill_bottle/paracetamol
+	name = "bottle of Paracetamol pills"
+	desc = "Contains pills used to relieve pain but better than Tramadol!!."
```
Comment (Anonmare) [inline code/game/objects/items/weapons/storage/firstaid.dm]: Consider losing the "!." to make it grammatically correct, a period after an exclamation point is awkward and two exclamation points seems unnecessary.

**Comment by Anonmare** [comment]
Consider swapping the names around, paracetamol and tramadol are real life painkillers and people will itnriniscally expect tramadol to be superior to paracetamol.

**Comment by Urgan123** [comment]
Right new plan im going to revert some changes and change paras and tramadols names around and fix the descriptions. For oxycodone im changing it back as it used to be if thats fine with you.

**Comment by TorinoFermic** [comment]
Well, all files with "nice" got replaced by tramadols

**Comment by Urgan123** [comment]
Yea not ideal im going to try and fix it

**Comment by Urgan123** [comment]
Now i pray

**Pull request closed without merging**